Build/Bloodmasters.cfg

```diff
@@ -73,6 +73,7 @@ windowwidth = 692;
 
 servermaps
 {
+	cemetery;
```

Comment:
Please avoid changing these configuration files. I am planning to get rid of them anyway.

If you want to do anything with a local config, please use `launchSettings.json`-provided configuration and the corresponding `Source/Client/server.debug.cfg` file, which already have a map.